src/app/core/config/app-config-properties.ts

```diff
@@ -40,6 +40,11 @@ export const AppConfigProperties = {
    */
   managementApiKey: 'EDC_UI_MANAGEMENT_API_KEY',
 
+  /**
+   * Connector Endpoint URL
+   */
+  connectorEndpoint: 'EDC_UI_CONNECTOR_ENDPOINT',
```

Comment:
Now I see this, and how in the Dashboard we get the information from the backend, it might sense to calculate the "isOwnConnector" flag in the backend...

Reply:
Backend tasks
- add isOwnEndpoint in UiAsset model in backend
- update catalog endpoint and use SelfDescriptionService to set isOwnEndpoint correctly